src/Controllers/Version2/class-wc-rest-system-status-v2-controller.php

```diff
@@ -72,16 +72,11 @@ public function get_items_permissions_check( $request ) {
 	 */
 	public function get_items( $request ) {
 		$schema   = $this->get_item_schema();
-		$mappings = $this->get_item_mappings();
+		$fields   = $this->get_fields_for_response( $request );
+		$mappings = $this->get_item_mappings( $fields );
 		$response = array();
 
 		foreach ( $mappings as $section => $values ) {
-			foreach ( $values as $key => $value ) {
```

Comment:
This code is just bad, trying to force the type based on the schema, but we already ensure it for each value returned already, also this code breaks the `_fields` behavior forcing to all fields get returned in the response.

Reply:
Done.

Reply:
> Or do you mean that get_item_mappings already returns the correctly typed values?
Yes.